src/lib/bindings/http/validation/structured.js

```diff
@@ -32,17 +32,20 @@ function parse(payload, headers, receiver) {
     source: undefined
   };
 
+
+  const unprocessed = new Set(Object.keys(incoming));
```

Comment:
I think the most efficient would be using the spread operator like this.
```suggestion
  const unprocessed = { ...incoming };
```